parent/pom.xml

```diff
@@ -146,6 +146,16 @@
             </dependency>
             <!-- END karaf version overrides -->
 
+
+            <dependency>
+                <groupId>org.codehaus.groovy</groupId>
+                <artifactId>groovy-all</artifactId>
+                <version>${groovy.version}</version>
+                <type>pom</type>
```

Comment:
Can you clarify what this change does? If a brooklyn module includes the dependency `groovy-all` will it still pull in all of the transitive dependencies of `groovy-all`?